docs/section10/10-14.md

```diff
@@ -0,0 +1,74 @@
+# Airside systems – Rule 10-14  
+**Schema Version:** 0.0.13    
+**Mandatory Rule:** True          
+**Rule ID:** 10-14  
+ 
+**Rule Description:** Baseline shall be modeled with the heating HVAC system efficiency per Tables G3.5.1-G3.5.6.  Where multiple HVAC zones or residential spaces are combined into a single thermal block the heating efficiencies (for baseline HVAC System Types 3 and 4) shall be based on the  equipment capacity of the thermal block divided by the number of HVAC zones or residential spaces.   
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1.2.1 Equipment Efficiencies  
+**Appendix G Section Reference:** None  
+**Data Lookup:** Tables G3.5.2, G3.5.4 and G3.5.5  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. Applicable to baseline HVAC system types 2, 3, 3a, 4, and 9.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()  
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- For each hvac system in the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`
```

Comment:
similar to #371 #375 and #376 need a case with NA outcome.

Reply:
as discussed during the RDS  call, the RCT applies the applicability check before the 'rule outcome' step and the NA outcome is not required in the RDS for that check to happen in the RCT